acceptance/auth/bundle_and_profile/output.txt

```diff
@@ -11,9 +11,9 @@
 >>> errcode [CLI] current-user me -t dev -p DEFAULT
 "[USERNAME]"
 
-=== Inside the bundle, profile flag not matching bundle host. Badness: should use profile from flag instead and not fail
+=== Inside the bundle, profile flag not matching bundle host.
```

Comment:
```suggestion
=== Inside the bundle, profile flag not matching bundle host. Should use profile from the flag and not the bundle.
```

Otherwise on a naive read it seems like `profile flag not matching bundle host` causes an error.